website/content/docs/upgrading/upgrade-to-1.8.0.mdx

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.8.0 - Guides
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.8.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.8.0 compared to 1.7. Please read it carefully.
+
+## License Enhancements
+
+Licenses and EULA enhancements have been introduced in the Vault 1.8 release.
+These changes are important for Enterprise customer to review. They do not affect
+OSS users. Please see the [License](docs/enterprise/license) documentation for more details.
+
+## Deprecations
+
+The following API endpoints have been deprecated and will be removed in a future release:
+
+* `PUT sys/license` to store licenses in storage; it is recommended to use
+   [License Autoloading](/docs/enterprise/license/autoloading) instead.
+
+* `/gcp/token/:roleset` and `/gcp/key/:roleset` paths for generating secrets for rolesets
```

Comment:
Should we add a note to ask users to update ACL policies accordingly?

Reply:
IMO there are potentially a lot of things that you might need to do to switch over (both in and outside of vault). There isn't anything really specially about ACL policies for this change so I'd be inclined to not call it out.